Sources/OpenTelemetrySdk/Metrics/Stable/LongGaugeBuilderSdk.swift

```diff
@@ -34,7 +34,13 @@ public class LongGaugeBuilderSdk : LongGaugeBuilder, InstrumentBuilder {
         registerLongAsynchronousInstrument(type: type, updater: callback)
     }
     
-
-    
+    public func setDescription(_ description: String) -> LongGaugeBuilder {
```

Comment:
setDescription & setUnit are accessible via the InstrumentBuilder protocol, remove these.